src/main/java/net/sf/jsqlparser/util/deparser/ExpressionDeParser.java

```diff
@@ -367,7 +374,7 @@ public void visit(SubSelect subSelect) {
         if (selectVisitor != null) {
             if (subSelect.getWithItemsList() != null) {
                 buffer.append("WITH ");
-                for (Iterator<WithItem> iter = subSelect.getWithItemsList().iterator(); iter.hasNext();) {
+                for (Iterator<WithItem> iter = subSelect.getWithItemsList().iterator(); iter.hasNext(); ) {
```

Comment:
This reformatting introduced a lot of changes. Reformating should be done in a different PR.

Reply:
Agreed.